include/zephyr/usb_c/usbc.h

```diff
@@ -220,6 +241,105 @@ typedef uint32_t (*policy_cb_get_rdo_t)(const struct device *dev);
  */
 typedef bool (*policy_cb_is_snk_at_default_t)(const struct device *dev);
 /** @}*/
+/** \addtogroup source_callbacks
+ *  @{
+ */
+/**
+ * @brief Callback type used to get the Rp value that should be placed on
+ *        the CC lines
+ *
+ * @param dev dev Runtime device structure
```

Comment:
```suggestion
 * @param dev USB-C Connector Instance
```

Replace all instances for the APIs your are introducing in this commit.

Reply:
done